Flash-Chat/Flash Chat iOS13/Controllers/ChatViewController.swift

```diff
@@ -0,0 +1,120 @@
+//
+//  ChatViewController.swift
+//  Flash Chat iOS13
+//
+//  Created by Angela Yu on 21/10/2019.
+//  Copyright © 2019 Angela Yu. All rights reserved.
+//
+
+import UIKit
+import Firebase
+
+class ChatViewController: UIViewController {
+
+    @IBOutlet weak var tableView: UITableView!
+    @IBOutlet weak var messageTextfield: UITextField!
+    
+    let db = Firestore.firestore()
+    
+    var messages: [Message] = []
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
+        tableView.dataSource = self
+        title = K.appName
+        navigationItem.hidesBackButton = true
+        
+        tableView.register(UINib(nibName: K.cellNibName, bundle: nil), forCellReuseIdentifier: K.cellIdentifier)
+        
+        loadMessages()
+    }
+    
+    func loadMessages() {
```

Comment:
チーム開発の場合は、他のクラスから呼び出しを想定舌メソッドか否か示すため、メソッドの可視性を意識します。

```suggestion
    private func loadMessages() {
```